composer.json

```diff
@@ -4,15 +4,15 @@
     "type": "project",
     "license": "GPL-2.0-or-later",
     "require": {
-        "php": ">=5.6",
+        "php": ">=8.2",
```

Comment:
composer.json requires php >=8.2 but your php version (7.2.34) does not satisfy that requirement.
php-codingstandards-codeigniter 1.0.1 requires php >=7.3 -> your php version (7.2.34) does not satisfy that requirement.

Reply:
> Why change this value? I would prefer to stay with >=5.6 for now since it still works with it. It is likely that the prebuilt packages for each php version since 5.6 won't work anymore... Let's see the outcome of the CI jobs.

A lot has changed with 8.2 and 8.3

Reply:
I already have the whole thing running actively on a php 8.3 system